### PR TITLE
Show units in size filter

### PR DESCRIPTION
**Pull request by fiskus**
# Description

# TODO

- [ ] Unit tests
- [ ] Automated tests (e.g. Preflight)
- [ ] Confirm that this change meets security best practices and does not violate the security model
- [ ] Documentation
    - [ ] [Python: Run `build.py`](../gendocs/build.py) for new docstrings
    - [ ] JavaScript: basic explanation and screenshot of new features
    - [ ] Markdown somewhere in docs/**/*.md that explains the feature to end users (said .md files should be linked from SUMMARY.md so they appear on https://docs.quiltdata.com)
    - [ ] Markdown docs for developers
- [ ] [Changelog](CHANGELOG.md) entry (skip if change is not significant to end users, e.g. docs only)

### CODE REVIEW

File: catalog/app/containers/Search/Search.tsx [hunk 1]
```diff
@@ -323,7 +337,7 @@ function FilterWidget(props: FilterWidgetProps) {
 
 const packageFilterLabels = {
   modified: 'Last modified',
-  size: 'Cumulative package size',
+  size: 'Cumulative package size, in bytes',
```
Comment (nl0) [inline catalog/app/containers/Search/Search.tsx:340]: i don't think we need this clarification if we display units

File: catalog/app/components/Filters/NumbersRange.tsx [hunk 2]
```diff
@@ -7,8 +7,8 @@ import { formatQuantity } from 'utils/string'
 const isNumber = (v: unknown): v is number => typeof v === 'number' && !Number.isNaN(v)
 
 const valueLabelFormat = (number: number) =>
-  // @ts-expect-error
   formatQuantity(number, {
+    fallback: 'Not a number',
     suffixes: ['', 'K', 'M', 'B', 'T', 'P', 'E', 'Z', 'Y'],
```
[the remark this answers is not in the document]
Comment (fiskus) [reply catalog/app/components/Filters/NumbersRange.tsx:12]: K - thousands, M - millions, B - billions, the rest is wrong and unchanged from bytes scales

[the remark this answers is not in the document]
Comment (nl0) [reply catalog/app/components/Filters/NumbersRange.tsx:12]: oh wow that's not what i expected, but ok, it makes some sense

**Comment by fiskus** [comment]
This PR was for demonstration purposes

**Pull request closed without merging**